Automattic/jetpack
# Social: Add upgrade trigger to Jetpack Social admin page

#### Changes proposed in this Pull Request:

* Add support for a link variant to the `ContextualUpgradeTrigger` component.
* Add support for passing a `className` to the `ContextualUpgradeTrigger` component.
* Add the upgrade trigger to the Jetpack Social admin page.

#### Other information:


#### Jetpack product discussion


p1662556927169929-slack-C02M27K4741
#### Does this pull request change what data or activity we track or use?


No.
#### Testing instructions:






To test the component:
* Make sure the component looks good in the Storybook:
  * `cd projects/js-packages/storybook && npm run storybook:dev`
* Check that the second story correctly opens the Jetpack page in a new tab.

To test the changes to Jetpack Social:
* Enable sharing limits for your site (first line in test plan of D85993-code)
* Activate Jetpack Social and open the admin page.
* Make sure the upgrade trigger is there, and that it opens the Jetpack Social landing page when clicked.
<img width="589" alt="Screenshot 2022-09-08 at 18 36 39@2x" src="https://user-images.githubusercontent.com/1713699/189177522-46c41b07-47e7-409a-b382-a9dd99842c1c.png">
